packages/fxa-auth-client/test/client.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as assert from 'assert';
+import AuthClient from '../server';
+
+describe('lib/client', () => {
+  let client: AuthClient;
+
+  before(() => {
+    client = new AuthClient('http://localhost:9000', {
```

Comment:
Wasn't sure if there was an env var or something we could use for these, but maybe we aren't testing against stage/prod and it doesn't matter; and it looks like we're hardcoding `localhost:9000` (and `127.0.0.1`) in a number of places already throughout the repo. 🤷 

```sh
git grep -Pn "(localhost|127\.0\.0\.1):9000" | wc -l # 60

git grep -Pho "(localhost|127\.0\.0\.1):9000" | sort | uniq -c | sort -nr
  58 localhost:9000
   2 127.0.0.1:9000
```

Reply:
Yeah, there are just minimal tests here. So I think this fine for now... It's surprising this class doesn't have test coverage already. I've filed a [follow up](https://mozilla-hub.atlassian.net/browse/FXA-9681) for this since it's been bugging me for awhile.